src/to-have-attribute.js

```diff
@@ -36,13 +42,14 @@ export function toHaveAttribute(htmlElement, name, expectedValue) {
           comment: getAttributeComment(name, expectedValue),
         },
       )
-      return getMessage(
+      return [
         matcher,
-        `Expected the element ${to} have attribute`,
-        printAttribute(name, expectedValue),
-        'Received',
-        receivedAttribute,
-      )
+        '',
+        `Expected the element ${to} have attribute:`,
+        `  ${expectedColor(printAttribute(name, expectedValue))}`,
+        'Received:',
+        `  ${receivedColor(receivedAttribute)}`,
+      ].join('\n')
```

Comment:
This is lazy fix for the case when `receivedAttribute` is `null`. Out of the box stringify in `getMessage` will add quotes for already strings with escaping e.t.c.  

Now i'm starting to think about removing 'Received:' part completely in such cases, but not so sure.

Reply:
Sorry for insisting, but what do you refer to with `stringify in getMessage`? I don't see stringify being used in it.

I insist because in every case we can get away with using `getMessage` is better, in case we change that format of messages we have a centralized way of doing it.

Reply:
Thank you very much!

Reply:
Fixed.You’re welcome! :bowtie: